app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBAdvancedSettings.tsx

```diff
@@ -1,25 +1,40 @@
 import React from 'react';
 
+import { FormikProps } from 'formik';
 import { FormikFormField, CheckboxInput, NumberInput, HelpField } from '@spinnaker/core';
+import { IAmazonApplicationLoadBalancerUpsertCommand } from 'amazon/domain';
 
-export class ALBAdvancedSettings extends React.Component {
-  public render() {
-    return (
-      <div>
-        <FormikFormField
-          name="idleTimeout"
-          label="Idle Timeout"
-          help={<HelpField id="loadBalancer.advancedSettings.idleTimeout" />}
-          input={(props) => <NumberInput {...props} min={0} />}
-        />
+export interface IALBAdvancedSettingsProps {
+  formik: FormikProps<IAmazonApplicationLoadBalancerUpsertCommand>;
+}
+
+export const ALBAdvancedSettings = React.forwardRef<HTMLDivElement, IALBAdvancedSettingsProps>((props, ref) => (
```

Comment:
I'm questioning if `forwardRef` is necessary at all, since it's being applied on the root jsx element anyway.

Reply:
It is needed for its use as a `WizardPage` in order for me to pass the ref into the root element.

Reply:
ah, gotcha!